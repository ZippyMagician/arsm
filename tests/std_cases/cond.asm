.loop
    mov ax in
    cz ax
    chl 0

    ceq ax '0
    cmo ax '1
    chr ax
    jmp :loop