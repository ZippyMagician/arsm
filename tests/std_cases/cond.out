111234111123411111123